tests/server/jest.config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  coverageDirectory: 'coverage',
+  testEnvironment: 'node',
+  preset: 'ts-jest',
+  roots: [
+    '<rootDir>/src'
+  ],
+  testMatch: ['**/?(*.)+(spec|test).[jt]s?(x)'] 
+}
```

Comment:
This will not work for CI. Please take `jest.config.js` from `platform`

Reply:
Fixed.